### PR TITLE
Build an operator for SORMAS

### PR DESCRIPTION
**Pull request by nk-designz**
## Changes
* the operator is build with the redhat operator framework and runs well on OpenShift and Kubernetes
* added a readme with an image of sormas on OpenShift

### CODE REVIEW

**Comment by enyachoke** [comment]
Hi @nk-designz thanks for the contribution. Please fix the conflicts and Squash the commits into one. Looking forward to testing the operator.

Also include in the readme Description on how to deploy the operator on openshift and kubernetes.

File: sormas-operator/deploy/crds/sormas.hzi-braunschweig.com_v1alpha1_sormashelm_cr.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+apiVersion: sormas.hzi-braunschweig.com/v1alpha1
+kind: SormasHelm
+metadata:
+  name: example-sormashelm
```
Comment (enyachoke) [inline sormas-operator/deploy/crds/sormas.hzi-braunschweig.com_v1alpha1_sormashelm_cr.yaml:4]: Clean up this bit

Comment (nk-designz) [reply sormas-operator/deploy/crds/sormas.hzi-braunschweig.com_v1alpha1_sormashelm_cr.yaml:4]: Which part of it exactly?

Comment (nk-designz) [reply sormas-operator/deploy/crds/sormas.hzi-braunschweig.com_v1alpha1_sormashelm_cr.yaml:4]: > Hi @nk-designz thanks for the contribution. Please fix the conflicts and Squash the commits into one. Looking forward to testing the operator.
> 
> Also include in the readme Description on how to deploy the operator on openshift and kubernetes.

* The conflict is my updated readme.
* The instructions are in the readme
* The commits are squashed like a mosquito

And the ```..._cr.yaml``` is a autogenerated sample by the operator-sdk for a test deployment of SORMAS

Have fun with it :D

Comment (nk-designz) [reply sormas-operator/deploy/crds/sormas.hzi-braunschweig.com_v1alpha1_sormashelm_cr.yaml:4]: Now the conflict is resolved.
I wondered how to do that.

Comment (enyachoke) [reply sormas-operator/deploy/crds/sormas.hzi-braunschweig.com_v1alpha1_sormashelm_cr.yaml:4]: Will merge and test it out tomorrow. Almost  12am here. Thanks for the good work.

**Pull request merged**